contribs/src/main/java/com/netflix/conductor/contribs/http/HttpTask.java

```diff
@@ -242,8 +245,10 @@ public void start(Workflow workflow, Task task, WorkflowExecutor executor) throw
 					task.getTaskDefName(),
 					serviceName,
 					exec_time);
+		}finally{
+			if (scheduledFuture != null && !scheduledFuture.isDone() && !scheduledFuture.isCancelled())
+				scheduledFuture.cancel(true);
```

Comment:
During normal execution, the scheduleFuture will be completed.. The cancel(true) is to cancel any futures that are currently scheduled since we dont want to update the queue item after HTTP is completed..
In case of an exception or shutdown, we want to forcefully terminate the scheduleFuture